protocol/daemons/flags/flags.go

```diff
@@ -127,6 +130,11 @@ func AddDaemonFlagsToCmd(
 		df.Liquidation.SubaccountPageLimit,
 		"Limit on the number of subaccounts to fetch per query in the Liquidation Daemon task loop.",
 	)
+	cmd.Flags().Uint64(
+		FlagLiquidationDaemonRequestChunkSize,
```

Comment:
Is there any logic that validates this value?

Specifically if it's easy, would be nice to validate this is non-zero.

Reply:
I don't think we validate any of the flags, so going to leave it as is for now. Created this [ticket](https://linear.app/dydx/issue/CLOB-904/validate-appdaemonclob-flags) to keep track